### ISSUE TITLE
[Accessibility] Selected key signature not announced in New Score dialog (macOS)

### ISSUE THREAD

**Issue by shoogle** [musescore/MuseScore#13599]
**Describe the bug**
In the New Score dialog's Key Signature chooser popup, screen readers announce which key signature has focus but they do not announce whether or not it is selected.

**To Reproduce**
1. File > New
2. Pick any template. Next.
3. Expand the **Key signature** popup.
4. Navigate the popup with arrow keys and tab.

**Expected behavior**
The screen reader announces the name of the key signature that has focus. It should also do either/both of the following:

- Say "selected" or "checked" if the focussed key signature is also the selected one.
- Say "not selected" or "not checked" when any non-selected key signature has focus.

It's not necessary to do both of these things as long as the screen reader does one of them correctly and just says the name for the other key signatures. However, currently NVDA says "not selected" for all key signatures and Narrator just says the name for all of them.

**Screenshots**

The selected key signature is highlighted in blue. The focussed one has a black outline.
<img width="600px" src="https://user-images.githubusercontent.com/11011881/193600743-c4eba246-ba25-457e-9cd3-38540b8fbb6f.png"/>

**Platform information**
 - OS: Windows 10 running NVDA or Narrator screen readers

**Additional context**
Speech does work for the note chooser in the Tempo popup, so the key signature chooser could be implemented in the same way.
<img width="500px" src="https://user-images.githubusercontent.com/11011881/193603094-898d460d-8137-478c-b8e7-cfe65849304d.png"/>

**Comment by DmitryArefiev** [comment]
@shoogle @Tantacrul Moving to Beta2 since it's not critical. Please return back if disagree.

**Comment by shubham-shinde-442** [comment]
You can close this issue, because as mentioned in issue description, screen readers is already announcing one of the state correctly.

**Comment by shoogle** [comment]
I've confirmed that key signatures in the New Score dialog are spoken by NVDA and Narrator on Windows.

If someone on macOS can confirm with VoiceOver (shortcut: Cmd+F5) then I think we can close this.

**Comment by DmitryArefiev** [comment]
@shoogle Tried on my side. Looks good on Windows and Narrator (was fixed in 4.2.0). 
But **still occurs** on **Mac** (4.4 master on macOS13.6)


https://github.com/musescore/MuseScore/assets/90187801/aeca9ae4-a095-4336-af70-4d6be202daac
